markets/perps-market/contracts/storage/GlobalPerpsMarket.sol

```diff
@@ -73,13 +73,14 @@ library GlobalPerpsMarket {
         Data storage self
     ) internal view returns (uint128 rate, uint256 delegatedCollateralValue, uint256 lockedCredit) {
         uint256 withdrawableUsd = PerpsMarketFactory.totalWithdrawableUsd();
-        delegatedCollateralValue = withdrawableUsd - totalCollateralValue(self);
+        int256 delegatedCollateralValueAsInt = withdrawableUsd.toInt() -
+            totalCollateralValue(self).toInt();
```

Comment:
this number `totalWithdrawableUsd` is comprised of `creditCapacity`  which is basically
1) lp's delegated collateral 
2) pnl (i.e. debt)

PLUS  `getDepositedCollateralValue,` which is all  traders' margin (aka collaterals)

Now this probably means 
We can just pull `creditCapacity` since `delegatedCollateralValueAsInt`  is  just recalculating something that we already have?
also how do i get a read on this `totalWithdrawableUsd` in cannon, can't seem to be able to find this function in PerpMarketProxy